2023-03-04 06:35:15,544:[critical]	./LimbusLocalize/assets/Localize/CN/CN_AbDlg_Rodion.json
	Traceback (most recent call last):
  File "/home/runner/work/Limbus-Company-Work-Dataset/Limbus-Company-Work-Dataset/run.py", line 119, in main
    last = check(merge(fetch_json(i)), last)
  File "/home/runner/work/Limbus-Company-Work-Dataset/Limbus-Company-Work-Dataset/run.py", line 105, in check
    if is_Chinese(CN):
  File "/home/runner/work/Limbus-Company-Work-Dataset/Limbus-Company-Work-Dataset/run.py", line 17, in is_Chinese
    for ch in word:
TypeError: 'int' object is not iterable

2023-03-04 06:35:15,544:[critical]	./LimbusLocalize/assets/Localize/CN/CN_Voice_EGO_YiSang_1.json
	Traceback (most recent call last):
  File "/home/runner/work/Limbus-Company-Work-Dataset/Limbus-Company-Work-Dataset/run.py", line 119, in main
    last = check(merge(fetch_json(i)), last)
  File "/home/runner/work/Limbus-Company-Work-Dataset/Limbus-Company-Work-Dataset/run.py", line 105, in check
    if is_Chinese(CN):
  File "/home/runner/work/Limbus-Company-Work-Dataset/Limbus-Company-Work-Dataset/run.py", line 17, in is_Chinese
    for ch in word:
TypeError: 'int' object is not iterable

2023-03-04 06:35:15,547:[critical]	./LimbusLocalize/assets/Localize/CN/CN_AbEventsResultLog.json
	Traceback (most recent call last):
  File "/home/runner/work/Limbus-Company-Work-Dataset/Limbus-Company-Work-Dataset/run.py", line 119, in main
    last = check(merge(fetch_json(i)), last)
  File "/home/runner/work/Limbus-Company-Work-Dataset/Limbus-Company-Work-Dataset/run.py", line 105, in check
    if is_Chinese(CN):
  File "/home/runner/work/Limbus-Company-Work-Dataset/Limbus-Company-Work-Dataset/run.py", line 17, in is_Chinese
    for ch in word:
TypeError: 'int' object is not iterable

2023-03-04 06:35:15,563:[critical]	./LimbusLocalize/assets/Localize/CN/CN_AbDlg_HongLu.json
	Traceback (most recent call last):
  File "/home/runner/work/Limbus-Company-Work-Dataset/Limbus-Company-Work-Dataset/run.py", line 119, in main
    last = check(merge(fetch_json(i)), last)
  File "/home/runner/work/Limbus-Company-Work-Dataset/Limbus-Company-Work-Dataset/run.py", line 105, in check
    if is_Chinese(CN):
  File "/home/runner/work/Limbus-Company-Work-Dataset/Limbus-Company-Work-Dataset/run.py", line 17, in is_Chinese
    for ch in word:
TypeError: 'int' object is not iterable

2023-03-04 06:35:15,563:[critical]	./LimbusLocalize/assets/Localize/CN/CN_Passives_Enemy.json
	Traceback (most recent call last):
  File "/home/runner/work/Limbus-Company-Work-Dataset/Limbus-Company-Work-Dataset/run.py", line 119, in main
    last = check(merge(fetch_json(i)), last)
  File "/home/runner/work/Limbus-Company-Work-Dataset/Limbus-Company-Work-Dataset/run.py", line 105, in check
    if is_Chinese(CN):
  File "/home/runner/work/Limbus-Company-Work-Dataset/Limbus-Company-Work-Dataset/run.py", line 17, in is_Chinese
    for ch in word:
TypeError: 'int' object is not iterable

2023-03-04 06:35:15,806:[critical]	./LimbusLocalize/assets/Localize/CN/CN_AbDlg_Gregor.json
	Traceback (most recent call last):
  File "/home/runner/work/Limbus-Company-Work-Dataset/Limbus-Company-Work-Dataset/run.py", line 119, in main
    last = check(merge(fetch_json(i)), last)
  File "/home/runner/work/Limbus-Company-Work-Dataset/Limbus-Company-Work-Dataset/run.py", line 105, in check
    if is_Chinese(CN):
  File "/home/runner/work/Limbus-Company-Work-Dataset/Limbus-Company-Work-Dataset/run.py", line 17, in is_Chinese
    for ch in word:
TypeError: 'int' object is not iterable

2023-03-04 06:35:15,806:[critical]	./LimbusLocalize/assets/Localize/CN/CN_Voice_EGO_Meursault_5.json
	Traceback (most recent call last):
  File "/home/runner/work/Limbus-Company-Work-Dataset/Limbus-Company-Work-Dataset/run.py", line 119, in main
    last = check(merge(fetch_json(i)), last)
  File "/home/runner/work/Limbus-Company-Work-Dataset/Limbus-Company-Work-Dataset/run.py", line 105, in check
    if is_Chinese(CN):
  File "/home/runner/work/Limbus-Company-Work-Dataset/Limbus-Company-Work-Dataset/run.py", line 17, in is_Chinese
    for ch in word:
TypeError: 'int' object is not iterable

2023-03-04 06:35:15,851:[ERROR]	./LimbusLocalize/assets/Localize/CN/CN_S108B.json load error
2023-03-04 06:35:15,853:[critical]	./LimbusLocalize/assets/Localize/CN/CN_S108B.json
	Traceback (most recent call last):
  File "/home/runner/work/Limbus-Company-Work-Dataset/Limbus-Company-Work-Dataset/run.py", line 74, in fetch_json
    fin[i] = json.load(f)
  File "/usr/lib/python3.10/json/__init__.py", line 293, in load
    return loads(fp.read(),
  File "/usr/lib/python3.10/json/__init__.py", line 346, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python3.10/json/decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python3.10/json/decoder.py", line 355, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 25 column 18 (char 405)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/runner/work/Limbus-Company-Work-Dataset/Limbus-Company-Work-Dataset/run.py", line 119, in main
    last = check(merge(fetch_json(i)), last)
  File "/home/runner/work/Limbus-Company-Work-Dataset/Limbus-Company-Work-Dataset/run.py", line 78, in fetch_json
    fin[i] = eval(f.read())
  File "<string>", line 25
    "content": 与迄今为止所撞见的人都不同，那些人的身上有一部分看起来就像昆虫一样。"
                            ^
SyntaxError: invalid character '，' (U+FF0C)

2023-03-04 06:35:16,016:[ERROR]	./LimbusLocalize/assets/Localize/CN/CN_S101B.json load error
2023-03-04 06:35:16,016:[critical]	./LimbusLocalize/assets/Localize/CN/CN_S101B.json
	Traceback (most recent call last):
  File "/home/runner/work/Limbus-Company-Work-Dataset/Limbus-Company-Work-Dataset/run.py", line 74, in fetch_json
    fin[i] = json.load(f)
  File "/usr/lib/python3.10/json/__init__.py", line 293, in load
    return loads(fp.read(),
  File "/usr/lib/python3.10/json/__init__.py", line 346, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python3.10/json/decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python3.10/json/decoder.py", line 353, in raw_decode
    obj, end = self.scan_once(s, idx)
json.decoder.JSONDecodeError: Invalid control character at: line 9 column 27 (char 126)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/runner/work/Limbus-Company-Work-Dataset/Limbus-Company-Work-Dataset/run.py", line 119, in main
    last = check(merge(fetch_json(i)), last)
  File "/home/runner/work/Limbus-Company-Work-Dataset/Limbus-Company-Work-Dataset/run.py", line 78, in fetch_json
    fin[i] = eval(f.read())
  File "<string>", line 9
    "place": "梅菲斯托费勒斯 内部
             ^
SyntaxError: unterminated string literal (detected at line 9)

2023-03-04 06:35:16,029:[critical]	./LimbusLocalize/assets/Localize/CN/CN_AbDlg_Merusault.json
	Traceback (most recent call last):
  File "/home/runner/work/Limbus-Company-Work-Dataset/Limbus-Company-Work-Dataset/run.py", line 119, in main
    last = check(merge(fetch_json(i)), last)
  File "/home/runner/work/Limbus-Company-Work-Dataset/Limbus-Company-Work-Dataset/run.py", line 105, in check
    if is_Chinese(CN):
  File "/home/runner/work/Limbus-Company-Work-Dataset/Limbus-Company-Work-Dataset/run.py", line 17, in is_Chinese
    for ch in word:
TypeError: 'int' object is not iterable

2023-03-04 06:35:16,224:[ERROR]	./LimbusLocalize/assets/Localize/CN/CN_S204B.json load error
2023-03-04 06:35:16,224:[critical]	./LimbusLocalize/assets/Localize/CN/CN_S204B.json
	Traceback (most recent call last):
  File "/home/runner/work/Limbus-Company-Work-Dataset/Limbus-Company-Work-Dataset/run.py", line 74, in fetch_json
    fin[i] = json.load(f)
  File "/usr/lib/python3.10/json/__init__.py", line 293, in load
    return loads(fp.read(),
  File "/usr/lib/python3.10/json/__init__.py", line 346, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python3.10/json/decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python3.10/json/decoder.py", line 353, in raw_decode
    obj, end = self.scan_once(s, idx)
json.decoder.JSONDecodeError: Invalid control character at: line 61 column 47 (char 1194)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/runner/work/Limbus-Company-Work-Dataset/Limbus-Company-Work-Dataset/run.py", line 119, in main
    last = check(merge(fetch_json(i)), last)
  File "/home/runner/work/Limbus-Company-Work-Dataset/Limbus-Company-Work-Dataset/run.py", line 78, in fetch_json
    fin[i] = eval(f.read())
  File "<string>", line 61
    "content": "你们两个，直到现在都觉得我们是毫无价值的人渣吧，不是吗？
               ^
SyntaxError: unterminated string literal (detected at line 61)

2023-03-04 06:35:16,328:[critical]	./LimbusLocalize/assets/Localize/CN/CN_AbDlg_YiSang.json
	Traceback (most recent call last):
  File "/home/runner/work/Limbus-Company-Work-Dataset/Limbus-Company-Work-Dataset/run.py", line 119, in main
    last = check(merge(fetch_json(i)), last)
  File "/home/runner/work/Limbus-Company-Work-Dataset/Limbus-Company-Work-Dataset/run.py", line 105, in check
    if is_Chinese(CN):
  File "/home/runner/work/Limbus-Company-Work-Dataset/Limbus-Company-Work-Dataset/run.py", line 17, in is_Chinese
    for ch in word:
TypeError: 'int' object is not iterable

2023-03-04 06:35:16,331:[critical]	./LimbusLocalize/assets/Localize/CN/CN_AbnormalityGuides.json
	Traceback (most recent call last):
  File "/home/runner/work/Limbus-Company-Work-Dataset/Limbus-Company-Work-Dataset/run.py", line 119, in main
    last = check(merge(fetch_json(i)), last)
  File "/home/runner/work/Limbus-Company-Work-Dataset/Limbus-Company-Work-Dataset/run.py", line 105, in check
    if is_Chinese(CN):
  File "/home/runner/work/Limbus-Company-Work-Dataset/Limbus-Company-Work-Dataset/run.py", line 17, in is_Chinese
    for ch in word:
TypeError: 'int' object is not iterable

2023-03-04 06:35:16,331:[critical]	./LimbusLocalize/assets/Localize/CN/CN_ShopItemCount.json
	Traceback (most recent call last):
  File "/home/runner/work/Limbus-Company-Work-Dataset/Limbus-Company-Work-Dataset/run.py", line 119, in main
    last = check(merge(fetch_json(i)), last)
  File "/home/runner/work/Limbus-Company-Work-Dataset/Limbus-Company-Work-Dataset/run.py", line 105, in check
    if is_Chinese(CN):
  File "/home/runner/work/Limbus-Company-Work-Dataset/Limbus-Company-Work-Dataset/run.py", line 17, in is_Chinese
    for ch in word:
TypeError: 'int' object is not iterable

2023-03-04 06:35:16,331:[critical]	./LimbusLocalize/assets/Localize/CN/CN_S0_13.json
	Traceback (most recent call last):
  File "/home/runner/work/Limbus-Company-Work-Dataset/Limbus-Company-Work-Dataset/run.py", line 119, in main
    last = check(merge(fetch_json(i)), last)
  File "/home/runner/work/Limbus-Company-Work-Dataset/Limbus-Company-Work-Dataset/run.py", line 105, in check
    if is_Chinese(CN):
  File "/home/runner/work/Limbus-Company-Work-Dataset/Limbus-Company-Work-Dataset/run.py", line 17, in is_Chinese
    for ch in word:
TypeError: 'int' object is not iterable

2023-03-04 06:35:16,332:[critical]	./LimbusLocalize/assets/Localize/CN/CN_StageNode1-3.json
	Traceback (most recent call last):
  File "/home/runner/work/Limbus-Company-Work-Dataset/Limbus-Company-Work-Dataset/run.py", line 119, in main
    last = check(merge(fetch_json(i)), last)
  File "/home/runner/work/Limbus-Company-Work-Dataset/Limbus-Company-Work-Dataset/run.py", line 105, in check
    if is_Chinese(CN):
  File "/home/runner/work/Limbus-Company-Work-Dataset/Limbus-Company-Work-Dataset/run.py", line 17, in is_Chinese
    for ch in word:
TypeError: 'int' object is not iterable

2023-03-04 06:35:16,333:[critical]	./LimbusLocalize/assets/Localize/CN/CN_AbDlg_Sinclair.json
	Traceback (most recent call last):
  File "/home/runner/work/Limbus-Company-Work-Dataset/Limbus-Company-Work-Dataset/run.py", line 119, in main
    last = check(merge(fetch_json(i)), last)
  File "/home/runner/work/Limbus-Company-Work-Dataset/Limbus-Company-Work-Dataset/run.py", line 105, in check
    if is_Chinese(CN):
  File "/home/runner/work/Limbus-Company-Work-Dataset/Limbus-Company-Work-Dataset/run.py", line 17, in is_Chinese
    for ch in word:
TypeError: 'int' object is not iterable

2023-03-04 06:35:16,333:[critical]	./LimbusLocalize/assets/Localize/CN/CN_StageNode1-1.json
	Traceback (most recent call last):
  File "/home/runner/work/Limbus-Company-Work-Dataset/Limbus-Company-Work-Dataset/run.py", line 119, in main
    last = check(merge(fetch_json(i)), last)
  File "/home/runner/work/Limbus-Company-Work-Dataset/Limbus-Company-Work-Dataset/run.py", line 105, in check
    if is_Chinese(CN):
  File "/home/runner/work/Limbus-Company-Work-Dataset/Limbus-Company-Work-Dataset/run.py", line 17, in is_Chinese
    for ch in word:
TypeError: 'int' object is not iterable

2023-03-04 06:35:16,366:[critical]	./LimbusLocalize/assets/Localize/CN/CN_AbDlg_Outis.json
	Traceback (most recent call last):
  File "/home/runner/work/Limbus-Company-Work-Dataset/Limbus-Company-Work-Dataset/run.py", line 119, in main
    last = check(merge(fetch_json(i)), last)
  File "/home/runner/work/Limbus-Company-Work-Dataset/Limbus-Company-Work-Dataset/run.py", line 105, in check
    if is_Chinese(CN):
  File "/home/runner/work/Limbus-Company-Work-Dataset/Limbus-Company-Work-Dataset/run.py", line 17, in is_Chinese
    for ch in word:
TypeError: 'int' object is not iterable

2023-03-04 06:35:16,410:[critical]	./LimbusLocalize/assets/Localize/CN/CN_AbDlg_Faust.json
	Traceback (most recent call last):
  File "/home/runner/work/Limbus-Company-Work-Dataset/Limbus-Company-Work-Dataset/run.py", line 119, in main
    last = check(merge(fetch_json(i)), last)
  File "/home/runner/work/Limbus-Company-Work-Dataset/Limbus-Company-Work-Dataset/run.py", line 105, in check
    if is_Chinese(CN):
  File "/home/runner/work/Limbus-Company-Work-Dataset/Limbus-Company-Work-Dataset/run.py", line 17, in is_Chinese
    for ch in word:
TypeError: 'int' object is not iterable

2023-03-04 06:35:16,411:[critical]	./LimbusLocalize/assets/Localize/CN/CN_S0_27.json
	Traceback (most recent call last):
  File "/home/runner/work/Limbus-Company-Work-Dataset/Limbus-Company-Work-Dataset/run.py", line 119, in main
    last = check(merge(fetch_json(i)), last)
  File "/home/runner/work/Limbus-Company-Work-Dataset/Limbus-Company-Work-Dataset/run.py", line 105, in check
    if is_Chinese(CN):
  File "/home/runner/work/Limbus-Company-Work-Dataset/Limbus-Company-Work-Dataset/run.py", line 17, in is_Chinese
    for ch in word:
TypeError: 'int' object is not iterable

2023-03-04 06:35:16,411:[critical]	./LimbusLocalize/assets/Localize/CN/CN_S210B.json
	Traceback (most recent call last):
  File "/home/runner/work/Limbus-Company-Work-Dataset/Limbus-Company-Work-Dataset/run.py", line 119, in main
    last = check(merge(fetch_json(i)), last)
  File "/home/runner/work/Limbus-Company-Work-Dataset/Limbus-Company-Work-Dataset/run.py", line 105, in check
    if is_Chinese(CN):
  File "/home/runner/work/Limbus-Company-Work-Dataset/Limbus-Company-Work-Dataset/run.py", line 17, in is_Chinese
    for ch in word:
TypeError: 'int' object is not iterable

2023-03-04 06:35:16,411:[critical]	./LimbusLocalize/assets/Localize/CN/CN_Voice_EGO_HongLu_6.json
	Traceback (most recent call last):
  File "/home/runner/work/Limbus-Company-Work-Dataset/Limbus-Company-Work-Dataset/run.py", line 119, in main
    last = check(merge(fetch_json(i)), last)
  File "/home/runner/work/Limbus-Company-Work-Dataset/Limbus-Company-Work-Dataset/run.py", line 105, in check
    if is_Chinese(CN):
  File "/home/runner/work/Limbus-Company-Work-Dataset/Limbus-Company-Work-Dataset/run.py", line 17, in is_Chinese
    for ch in word:
TypeError: 'int' object is not iterable

2023-03-04 06:35:16,412:[critical]	./LimbusLocalize/assets/Localize/CN/CN_Announcer.json
	Traceback (most recent call last):
  File "/home/runner/work/Limbus-Company-Work-Dataset/Limbus-Company-Work-Dataset/run.py", line 119, in main
    last = check(merge(fetch_json(i)), last)
  File "/home/runner/work/Limbus-Company-Work-Dataset/Limbus-Company-Work-Dataset/run.py", line 105, in check
    if is_Chinese(CN):
  File "/home/runner/work/Limbus-Company-Work-Dataset/Limbus-Company-Work-Dataset/run.py", line 17, in is_Chinese
    for ch in word:
TypeError: 'int' object is not iterable

2023-03-04 06:35:16,413:[critical]	./LimbusLocalize/assets/Localize/CN/CN_3D103B.json
	Traceback (most recent call last):
  File "/home/runner/work/Limbus-Company-Work-Dataset/Limbus-Company-Work-Dataset/run.py", line 119, in main
    last = check(merge(fetch_json(i)), last)
  File "/home/runner/work/Limbus-Company-Work-Dataset/Limbus-Company-Work-Dataset/run.py", line 105, in check
    if is_Chinese(CN):
  File "/home/runner/work/Limbus-Company-Work-Dataset/Limbus-Company-Work-Dataset/run.py", line 17, in is_Chinese
    for ch in word:
TypeError: 'int' object is not iterable

2023-03-04 06:35:16,526:[critical]	./LimbusLocalize/assets/Localize/CN/CN_NickName.json
	Traceback (most recent call last):
  File "/home/runner/work/Limbus-Company-Work-Dataset/Limbus-Company-Work-Dataset/run.py", line 119, in main
    last = check(merge(fetch_json(i)), last)
  File "/home/runner/work/Limbus-Company-Work-Dataset/Limbus-Company-Work-Dataset/run.py", line 84, in merge
    for i in fin["CN"]["dataList"]:
KeyError: 'dataList'

2023-03-04 06:35:16,931:[critical]	./LimbusLocalize/assets/Localize/CN/CN_3D309B.json
	Traceback (most recent call last):
  File "/home/runner/work/Limbus-Company-Work-Dataset/Limbus-Company-Work-Dataset/run.py", line 119, in main
    last = check(merge(fetch_json(i)), last)
  File "/home/runner/work/Limbus-Company-Work-Dataset/Limbus-Company-Work-Dataset/run.py", line 103, in check
    for j in out[i]["CN"]:
KeyError: 'CN'

2023-03-04 06:35:16,932:[critical]	./LimbusLocalize/assets/Localize/CN/CN_StoryText.json
	Traceback (most recent call last):
  File "/home/runner/work/Limbus-Company-Work-Dataset/Limbus-Company-Work-Dataset/run.py", line 119, in main
    last = check(merge(fetch_json(i)), last)
  File "/home/runner/work/Limbus-Company-Work-Dataset/Limbus-Company-Work-Dataset/run.py", line 103, in check
    for j in out[i]["CN"]:
KeyError: 'CN'

2023-03-04 06:35:16,934:[critical]	./LimbusLocalize/assets/Localize/CN/CN_Passives.json
	Traceback (most recent call last):
  File "/home/runner/work/Limbus-Company-Work-Dataset/Limbus-Company-Work-Dataset/run.py", line 119, in main
    last = check(merge(fetch_json(i)), last)
  File "/home/runner/work/Limbus-Company-Work-Dataset/Limbus-Company-Work-Dataset/run.py", line 103, in check
    for j in out[i]["CN"]:
KeyError: 'CN'

2023-03-04 06:35:16,935:[critical]	./LimbusLocalize/assets/Localize/CN/CN_Voice_Sinclair_LCB_11001.json
	Traceback (most recent call last):
  File "/home/runner/work/Limbus-Company-Work-Dataset/Limbus-Company-Work-Dataset/run.py", line 119, in main
    last = check(merge(fetch_json(i)), last)
  File "/home/runner/work/Limbus-Company-Work-Dataset/Limbus-Company-Work-Dataset/run.py", line 103, in check
    for j in out[i]["CN"]:
KeyError: 'CN'

2023-03-04 06:35:16,936:[critical]	./LimbusLocalize/assets/Localize/CN/CN_EGOgift_StoryDungeon.json
	Traceback (most recent call last):
  File "/home/runner/work/Limbus-Company-Work-Dataset/Limbus-Company-Work-Dataset/run.py", line 119, in main
    last = check(merge(fetch_json(i)), last)
  File "/home/runner/work/Limbus-Company-Work-Dataset/Limbus-Company-Work-Dataset/run.py", line 103, in check
    for j in out[i]["CN"]:
KeyError: 'CN'

2023-03-04 06:35:16,936:[ERROR]	./LimbusLocalize/assets/Localize/CN/CN_S102B.json load error
2023-03-04 06:35:16,937:[critical]	./LimbusLocalize/assets/Localize/CN/CN_S102B.json
	Traceback (most recent call last):
  File "/home/runner/work/Limbus-Company-Work-Dataset/Limbus-Company-Work-Dataset/run.py", line 74, in fetch_json
    fin[i] = json.load(f)
  File "/usr/lib/python3.10/json/__init__.py", line 293, in load
    return loads(fp.read(),
  File "/usr/lib/python3.10/json/__init__.py", line 346, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python3.10/json/decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python3.10/json/decoder.py", line 353, in raw_decode
    obj, end = self.scan_once(s, idx)
json.decoder.JSONDecodeError: Invalid control character at: line 7 column 30 (char 121)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/runner/work/Limbus-Company-Work-Dataset/Limbus-Company-Work-Dataset/run.py", line 119, in main
    last = check(merge(fetch_json(i)), last)
  File "/home/runner/work/Limbus-Company-Work-Dataset/Limbus-Company-Work-Dataset/run.py", line 78, in fetch_json
    fin[i] = eval(f.read())
  File "<string>", line 7
    "content": "不好意思！我有个问题！
               ^
SyntaxError: unterminated string literal (detected at line 7)

2023-03-04 06:35:16,976:[critical]	./LimbusLocalize/assets/Localize/CN/CN_AbDlg_Ishmael.json
	Traceback (most recent call last):
  File "/home/runner/work/Limbus-Company-Work-Dataset/Limbus-Company-Work-Dataset/run.py", line 119, in main
    last = check(merge(fetch_json(i)), last)
  File "/home/runner/work/Limbus-Company-Work-Dataset/Limbus-Company-Work-Dataset/run.py", line 105, in check
    if is_Chinese(CN):
  File "/home/runner/work/Limbus-Company-Work-Dataset/Limbus-Company-Work-Dataset/run.py", line 17, in is_Chinese
    for ch in word:
TypeError: 'int' object is not iterable

2023-03-04 06:35:16,976:[critical]	./LimbusLocalize/assets/Localize/CN/CN_AbDlg_Heathcliff.json
	Traceback (most recent call last):
  File "/home/runner/work/Limbus-Company-Work-Dataset/Limbus-Company-Work-Dataset/run.py", line 119, in main
    last = check(merge(fetch_json(i)), last)
  File "/home/runner/work/Limbus-Company-Work-Dataset/Limbus-Company-Work-Dataset/run.py", line 105, in check
    if is_Chinese(CN):
  File "/home/runner/work/Limbus-Company-Work-Dataset/Limbus-Company-Work-Dataset/run.py", line 17, in is_Chinese
    for ch in word:
TypeError: 'int' object is not iterable

2023-03-04 06:35:16,977:[critical]	./LimbusLocalize/assets/Localize/CN/CN_BattlePass_Mission.json
	Traceback (most recent call last):
  File "/home/runner/work/Limbus-Company-Work-Dataset/Limbus-Company-Work-Dataset/run.py", line 119, in main
    last = check(merge(fetch_json(i)), last)
  File "/home/runner/work/Limbus-Company-Work-Dataset/Limbus-Company-Work-Dataset/run.py", line 105, in check
    if is_Chinese(CN):
  File "/home/runner/work/Limbus-Company-Work-Dataset/Limbus-Company-Work-Dataset/run.py", line 17, in is_Chinese
    for ch in word:
TypeError: 'int' object is not iterable

2023-03-04 06:35:16,977:[critical]	./LimbusLocalize/assets/Localize/CN/CN_S215B.json
	Traceback (most recent call last):
  File "/home/runner/work/Limbus-Company-Work-Dataset/Limbus-Company-Work-Dataset/run.py", line 119, in main
    last = check(merge(fetch_json(i)), last)
  File "/home/runner/work/Limbus-Company-Work-Dataset/Limbus-Company-Work-Dataset/run.py", line 105, in check
    if is_Chinese(CN):
  File "/home/runner/work/Limbus-Company-Work-Dataset/Limbus-Company-Work-Dataset/run.py", line 17, in is_Chinese
    for ch in word:
TypeError: 'int' object is not iterable

2023-03-04 06:35:16,978:[critical]	./LimbusLocalize/assets/Localize/CN/CN_Voice_Gregor_LCB_11201.json
	Traceback (most recent call last):
  File "/home/runner/work/Limbus-Company-Work-Dataset/Limbus-Company-Work-Dataset/run.py", line 119, in main
    last = check(merge(fetch_json(i)), last)
  File "/home/runner/work/Limbus-Company-Work-Dataset/Limbus-Company-Work-Dataset/run.py", line 105, in check
    if is_Chinese(CN):
  File "/home/runner/work/Limbus-Company-Work-Dataset/Limbus-Company-Work-Dataset/run.py", line 17, in is_Chinese
    for ch in word:
TypeError: 'int' object is not iterable

2023-03-04 06:35:16,979:[critical]	./LimbusLocalize/assets/Localize/CN/CN_TutorialDesc0-4.json
	Traceback (most recent call last):
  File "/home/runner/work/Limbus-Company-Work-Dataset/Limbus-Company-Work-Dataset/run.py", line 119, in main
    last = check(merge(fetch_json(i)), last)
  File "/home/runner/work/Limbus-Company-Work-Dataset/Limbus-Company-Work-Dataset/run.py", line 105, in check
    if is_Chinese(CN):
  File "/home/runner/work/Limbus-Company-Work-Dataset/Limbus-Company-Work-Dataset/run.py", line 17, in is_Chinese
    for ch in word:
TypeError: 'int' object is not iterable

2023-03-04 06:35:16,980:[critical]	./LimbusLocalize/assets/Localize/CN/CN_3D307B.json
	Traceback (most recent call last):
  File "/home/runner/work/Limbus-Company-Work-Dataset/Limbus-Company-Work-Dataset/run.py", line 119, in main
    last = check(merge(fetch_json(i)), last)
  File "/home/runner/work/Limbus-Company-Work-Dataset/Limbus-Company-Work-Dataset/run.py", line 105, in check
    if is_Chinese(CN):
  File "/home/runner/work/Limbus-Company-Work-Dataset/Limbus-Company-Work-Dataset/run.py", line 17, in is_Chinese
    for ch in word:
TypeError: 'int' object is not iterable

2023-03-04 06:35:17,349:[critical]	./LimbusLocalize/assets/Localize/CN/CN_AbDlg_DonQuixote.json
	Traceback (most recent call last):
  File "/home/runner/work/Limbus-Company-Work-Dataset/Limbus-Company-Work-Dataset/run.py", line 119, in main
    last = check(merge(fetch_json(i)), last)
  File "/home/runner/work/Limbus-Company-Work-Dataset/Limbus-Company-Work-Dataset/run.py", line 105, in check
    if is_Chinese(CN):
  File "/home/runner/work/Limbus-Company-Work-Dataset/Limbus-Company-Work-Dataset/run.py", line 17, in is_Chinese
    for ch in word:
TypeError: 'int' object is not iterable

2023-03-04 06:35:17,350:[critical]	./LimbusLocalize/assets/Localize/CN/CN_Skills_Tutorial.json
	Traceback (most recent call last):
  File "/home/runner/work/Limbus-Company-Work-Dataset/Limbus-Company-Work-Dataset/run.py", line 119, in main
    last = check(merge(fetch_json(i)), last)
  File "/home/runner/work/Limbus-Company-Work-Dataset/Limbus-Company-Work-Dataset/run.py", line 105, in check
    if is_Chinese(CN):
  File "/home/runner/work/Limbus-Company-Work-Dataset/Limbus-Company-Work-Dataset/run.py", line 17, in is_Chinese
    for ch in word:
TypeError: 'int' object is not iterable

2023-03-04 06:35:17,351:[critical]	./LimbusLocalize/assets/Localize/CN/CN_Voice_Gregor_GCorp_11203.json
	Traceback (most recent call last):
  File "/home/runner/work/Limbus-Company-Work-Dataset/Limbus-Company-Work-Dataset/run.py", line 119, in main
    last = check(merge(fetch_json(i)), last)
  File "/home/runner/work/Limbus-Company-Work-Dataset/Limbus-Company-Work-Dataset/run.py", line 105, in check
    if is_Chinese(CN):
  File "/home/runner/work/Limbus-Company-Work-Dataset/Limbus-Company-Work-Dataset/run.py", line 17, in is_Chinese
    for ch in word:
TypeError: 'int' object is not iterable

2023-03-04 06:35:17,352:[critical]	./LimbusLocalize/assets/Localize/CN/CN_StagePartText.json
	Traceback (most recent call last):
  File "/home/runner/work/Limbus-Company-Work-Dataset/Limbus-Company-Work-Dataset/run.py", line 119, in main
    last = check(merge(fetch_json(i)), last)
  File "/home/runner/work/Limbus-Company-Work-Dataset/Limbus-Company-Work-Dataset/run.py", line 105, in check
    if is_Chinese(CN):
  File "/home/runner/work/Limbus-Company-Work-Dataset/Limbus-Company-Work-Dataset/run.py", line 17, in is_Chinese
    for ch in word:
TypeError: 'int' object is not iterable

2023-03-04 06:35:17,382:[critical]	./LimbusLocalize/assets/Localize/CN/CN_AbDlg_Ryoshu.json
	Traceback (most recent call last):
  File "/home/runner/work/Limbus-Company-Work-Dataset/Limbus-Company-Work-Dataset/run.py", line 119, in main
    last = check(merge(fetch_json(i)), last)
  File "/home/runner/work/Limbus-Company-Work-Dataset/Limbus-Company-Work-Dataset/run.py", line 105, in check
    if is_Chinese(CN):
  File "/home/runner/work/Limbus-Company-Work-Dataset/Limbus-Company-Work-Dataset/run.py", line 17, in is_Chinese
    for ch in word:
TypeError: 'int' object is not iterable

2023-03-04 06:35:17,383:[critical]	./LimbusLocalize/assets/Localize/CN/CN_EGOgift_MirrorDungeon.json
	Traceback (most recent call last):
  File "/home/runner/work/Limbus-Company-Work-Dataset/Limbus-Company-Work-Dataset/run.py", line 119, in main
    last = check(merge(fetch_json(i)), last)
  File "/home/runner/work/Limbus-Company-Work-Dataset/Limbus-Company-Work-Dataset/run.py", line 105, in check
    if is_Chinese(CN):
  File "/home/runner/work/Limbus-Company-Work-Dataset/Limbus-Company-Work-Dataset/run.py", line 17, in is_Chinese
    for ch in word:
TypeError: 'int' object is not iterable

